src/application/attributes.rs

```diff
@@ -127,6 +130,12 @@ pub struct Attributes {
     ///
     /// The default is an empty vector.
     pub initialization_scripts: Vec<String>,
+
+    /// A closure that will be executed when a file is dropped on the window.
+    ///
+    /// The default is `None`.
+    #[cfg(feature = "file-drop")]
+    pub file_drop_handler: Option<FileDropHandler>,
```

Comment:
Looks like we don't need it in `Attributes`. I think this could also remove the requirement of Clone trait which we might remove that Arc in `FileDropHandler`.